charts/secrets-store-csi-driver-provider-aws/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: secrets-store-csi-driver-provider-aws
-version: 0.3.11
+version: 0.4.0
```

Comment:
Can you bump that to 1.0.0? Semver makes no backwards compatibility guarantees before 1.0.0